### PR TITLE
addeb bob exercise

### PR DESCRIPTION
**Pull request by gabrielperales**

### CODE REVIEW

File: exercises/bob/.gitignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.DS_Store
```
Comment (stevejb71) [inline exercises/bob/.gitignore]: There's a top level gitignore, so I don't think this is needed

File: exercises/bob/src/Bob.re [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+let alphaChars = (input: string) : bool =>
+  [%bs.re "/[a-zA-Z-]/"] |> Js.Re.test(input);
+
+let question = (input: string) : bool =>
+  String.contains_from(input, String.length(input) - 1, '?');
+
+let analyseSpeech = (speech: string) : speechType => {
+  let trimmed = String.trim(speech);
+  if (String.length(trimmed) === 0) {
```
Comment (stevejb71) [inline exercises/bob/src/Bob.re]: Use ```==```. This is the value equality operator. ```===``` does reference equality which is less commonly useful.

**Comment by gabrielperales** [comment]
@stevejb71 done!, let me know if there is anything I should change/improve

**Comment by gabrielperales** [comment]
@stevejb71 I have updated the code and renamed the file `Bob.re` to `Example.re`.

**Comment by stevejb71** [comment]
Looks good, I'll merge

**Pull request merged**